
# Tools document summary

# Data pre-processing
## 1.Fastq
- [Cutadapt](https://cutadapt.readthedocs.io/en/stable/index.html)
- [Fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
> [Fastqc modules](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/)
- [seqtk](https://github.com/lh3/seqtk)

## 2.Fastq mapping
- [bwa](https://bio-bwa.sourceforge.net/bwa.shtml)
- [STAR](https://github.com/alexdobin/STAR)
> [Bam format](https://github.com/samtools/hts-specs/blob/master/SAMv1.pdf)
> [BAM tags](https://github.com/samtools/hts-specs/blob/master/SAMtags.pdf)
- [samtools](http://www.htslib.org/doc/samtools.html)

# scRNA-seq analysis tools
## 1.Intergration
### R:
- [R:Seurat](https://satijalab.org/seurat/index.html)
- [Harmony](https://github.com/immunogenomics/harmony)
### Python:
- [Python:Scanpy](https://scanpy.readthedocs.io/en/stable/)

## 2.Enrichment analysis
- [clusterProfiler](https://yulab-smu.top/biomedical-knowledge-mining-book/index.html)

## 3.Trajectories analysis
- [Monocle_1_2](https://cole-trapnell-lab.github.io/monocle-release/)
> [Monocle3](https://cole-trapnell-lab.github.io/monocle3/)

## 4.Cell communication analysis
- [CellphoneDB](https://cellphonedb.readthedocs.io/en/latest/#)
- [CellChat](https://github.com/jinworks/CellChat)

## 5.Transcriptional regulatory analysis
- [SCENIC](https://scenic.aertslab.org/)

## 6.Metabolism analysis
- [scMetabolism](https://github.com/wu-yc/scMetabolism)